proto/pbpeering/peering.proto

```diff
@@ -108,12 +108,16 @@ message PeeringTrustBundle {
   // PeerName associates the trust bundle with a peer.
   string PeerName = 2;
 
-  // Partition isolates the bundle from other trust bundles in separate partitions.
+  // Partition isolates the bundle from other trust bundles in separate local partitions.
   string Partition = 3;
 
   // RootPEMs holds ASN.1 DER encoded X.509 certificate data for the trust bundle.
   repeated string RootPEMs = 4;
 
+  // ExportedPartition references the remote partition of the peer
+  // which sent this trust bundle. Used for generating SpiffeIDs.
+  string ExportedPartition = 7;
```

Comment:
Feel free to renumber these to make the fields have the right order with respect to the numbers. We didn't ship this to GA yet.